lib/core/src/Cardano/Pool/DB/Model.hs

```diff
@@ -180,22 +192,60 @@ mReadStakeDistribution epoch db@PoolDatabase{distributions} =
     , db
     )
 
-mPutPoolRegistration :: SlotId -> PoolRegistrationCertificate -> ModelPoolOp ()
-mPutPoolRegistration sl registration db@PoolDatabase{owners,registrations} =
+mPutPoolRegistration
+    :: CertificatePublicationTime
+    -> PoolRegistrationCertificate
+    -> ModelPoolOp ()
+mPutPoolRegistration sp registration db =
```

Comment:
Given that the type is `CertificatePublicationTime`, we should probably also rename `sp` to `cpt` for consistency.

```suggestion
mPutPoolRegistration cpt registration db =
```

Reply:
Fixed in a75c879aabde68edcc1389a38a9e09b0bbdf17a7.